packages/x-components/src/views/home/Home.vue

```diff
@@ -213,6 +213,14 @@
               <p>You may be interested in these:</p>
             </div>
 
+            <template v-if="!$x.query.searchBox">
+              <h1 class="x-title1 x-margin--bottom-06">Discovery wall</h1>
```

Comment:
As the Discovery Wall concept I think is being deprecated...could you change the title?

```suggestion
              <h1 class="x-title1 x-margin--bottom-06">Brand Recommendations</h1>
```